src/components/Heatmap.js

```diff
@@ -1,13 +1,16 @@
+/* eslint-disable */
 import React, { useState } from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { v4 as uuidv4 } from 'uuid';
 import HeatmapHours from './HeatmapHours';
 import HeatmapDays from './HeatmapDays';
 import HeatmapTile from './HeatmapTile';
+import PostsTable from './PostsTable';
 
 function Heatmap({ posts }) {
   const [selected, setSelected] = useState();
+  const [table, setTable] = useState([]);
```

Comment:
You don't need a second state variable here. Also see the "Don't duplicate your data" point in the code review